examples/java/basic_example/src/main/java/com/qlik/sse/basicExample/PluginServerInterceptor.java

```diff
@@ -0,0 +1,69 @@
+package com.qlik.sse.basicExample;
```

Comment:
package names should be all lowercase (https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html)